enter_bug.cgi

```diff
@@ -438,6 +439,32 @@ my $format = $template->get_format("bug/create/create",
                                    scalar $cgi->param('format'), 
                                    scalar $cgi->param('ctype'));
 
+my %data;
+my $count = 0;
+foreach my $c (@{ $product->components }) {
+    next if not $c->is_active;
+    $data{components}[ $count ] = $c->name;
+    $data{comp_desc}[ $count ] = $c->description;
+    $data{initialowners}[ $count ] = $c->default_assignee->login;
+    my @flag_list;
+    foreach my $f (@{ $c->flag_types->{bug} }) {
+        push (@flag_list, $f->id);
+    }
+    foreach my $f (@{ $c->flag_types->{attachment} }) {
+        push (@flag_list, $f->id);
+    }
+    $data{flags}[ $count ] = join(',', @flag_list);
+    $count ++;
+    if ( Bugzilla->params->{'useqacontact'} ) {
+        $data{initialqacontacts}[ $count ] = $c->default_qa_contact->login
+    }
+    my @initial_cc_list;
+    foreach my $cc_user (@{ $c->flag_types->{bug} }) {
```

Comment:
I think you meant to say @{ $c->initial_cc }, not $c->flag_types. I corrected for that and the code seems
to work on first pass.